source/stylesheets/manifest.css.scss

```diff
@@ -1,13 +1,5 @@
 //= link govuk_template_jinja/assets/images/gov.uk_logotype_crown_invert_trans.png
 
-//= link govuk_template_jinja/assets/stylesheets/fonts/GDSTransportTabular-Bold.eot
-//= link govuk_template_jinja/assets/stylesheets/fonts/GDSTransportTabular-Light.eot
-//= link govuk_template_jinja/assets/stylesheets/fonts/GDSTransportWebsite-Bold.eot
-//= link govuk_template_jinja/assets/stylesheets/fonts/GDSTransportWebsite.eot
-
-//= link govuk_template_jinja/assets/stylesheets/fonts.css
-//= link govuk_template_jinja/assets/stylesheets/fonts-ie8.css
-
```

Comment:
Required otherwise this error happens:
```
Sprockets::FileNotFound: couldn't find file 'govuk_template_jinja/assets/stylesheets/fonts-ie8.css'
Checked in these paths:
  /usr/src/app/node_modules
  /usr/src/app/source/javascripts
  /usr/src/app/source/stylesheets
  /usr/src/app/source
  /usr/src/app/source/stylesheets/manifest.css.scss:4
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/resolve.rb:65:in `resolve!'
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/directive_processor.rb:399:in `resolve'
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/directive_processor.rb:300:in `process_link_directive'
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/directive_processor.rb:180:in `block in process_directives'
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/directive_processor.rb:178:in `each'
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/directive_processor.rb:178:in `process_directives'
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/directive_processor.rb:83:in `_call'
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/directive_processor.rb:68:in `call'
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:75:in `call_processor'
  /usr/local/bundle/gems/sprockets-3.7.2/lib/sprockets/processor_utils.rb:57:in `block in call_
```

Reply:
What is this manifest for?
If it's something important, it should be updated alongside the govuk_template_jinja package, or we can rollback the package update from #56